src/main/scala/com.socrata.geoexport/http/ExportService.scala

```diff
@@ -83,19 +87,20 @@ class ExportService(sodaClient: UnmanagedCuratedServiceClient) extends SimpleRes
   }
 
   private def getUpstreamLayers(req: HttpRequest,
-                                fxfs: Seq[String]): Either[HttpResponse, Seq[Response with Closeable]] = {
-    fxfs.map { fbf =>
-      val soql = req.queryParameter("query").getOrElse(s"select * limit ${Int.MaxValue}")
-      val copy = req.queryParameter("copy").getOrElse("published")
-      val context = req.queryParameter("context").getOrElse("{}")
+                                layers: Seq[LayerWithQuery]): Either[HttpResponse, Seq[Response with Closeable]] = {
+    val defaultSoql = s"select * limit ${Int.MaxValue}"
```

Comment:
The limit parameter is required (and explicitly as big as we can make it) to ensure that we don't fall back to a default limit?

Reply:
Not my addition, but yep, that is my understanding, is that if you don't specify a limit the NBE defaults to 10000 or something.